.drone.yml

```diff
@@ -52,45 +52,26 @@ steps:
    - push
    - pull_request
 
-#- name: Unit Tests and SonarQube Scanner
```

Comment:
Is the `Unit Tests and SonarQube Scanner` step commented out for a reason? e.g.:
- It was erroring when running the trivy scanner.

Reply:
THis was to clean out the pipeline, do we still need it hushed out for later use?

Reply:
ps: I didn't comment them out, I can leave them commented out if they are needed for later use,

Reply:
No, I thought the commented code was added in, but I did not see that it was deleted. 🤦 
- Please ignore my comment. It is my mistake.

Reply:
the TOLERATE env_var is done away with